### DISCUSSION TITLE
Images show in local, but not when deployed to GH pages

### DISCUSSION THREAD

**Discussion by Pismice** [imfing/hextra#408] [Q&A]
**Description**

When on local, images are displayed only when I put them in in the static/images folder, it does not work when I relative path when my image is in the same folder as my org file (yes I am using org and not markdown).

The problem is the image is display on localhost, but when uploaded to my  [github pages](https://pismice.github.io/HEIG_ZIG/docs/web/zap/), the path to the image is broken.

**Steps To Reproduce**
1. Clone my repo https://github.com/Pismice/HEIG_ZIG
2. Run with `hugo server --buildDrafts --disableFastRender`
3. See that the images are on localhost, but not on my GH pages

**Expected Behavior**

Images should be shown

**Actual Behavior**

Images are not shown

**Screenshots**

**Environment**

- Hugo Version: [e.g., 0.126.2]
- Browser/OS: [Brave]
- Theme Version: [Did not know how to find it, but I dont think it is relevant]

**Additional Context**

Maybe it comes from my github workflow:  https://github.com/Pismice/HEIG_ZIG/blob/main/.github/workflows/pages.yaml

**Comment by imfing** [comment]
I noticed the image URL was not quite correct, (`
https://pismice.github.io/images/i3.png` instead of `https://pismice.github.io/HEIG_ZIG/images/i3.png`).

![image](https://github.com/imfing/hextra/assets/5097752/4f51aa4a-20d9-4abe-9da0-fa36e72f1039)

I think it's because the org files are processed differently in Hugo so the [`render-images.html`](https://github.com/imfing/hextra/blob/main/layouts/_default/_markup/render-image.html) hook was not used.

**Comment by Pismice** [comment]
How can I fix that then ? Because if I modify my path from `[[/images/req_per_sec_graph.png]]` to `[[HEIG_ZIG/images/req_per_sec_graph.png]]` it is going to work on the website but not on localhost :/

**Comment by imfing** [comment]
you should be able to use the `--baseURL` option to run the localhost under a subpath, which simulates the github pages path

**Reply by Pismice** [reply]
I am sorry but I still didnt get how to make it work :/ what sould my baseURL be ?

**Comment by Pismice** [comment]
Any idea someone please :(

**Comment by Pismice** [comment]
because things like logo.svg are found with /HEIG_ZIG/logo.svg but not my personal pictures

**Reply by Pismice** [reply]
this is the new command I tried and it changed nothing when using in local:
`hugo server --buildDrafts --disableFastRender --baseURL HEIG_ZIG`

**Reply by imfing** [reply]
did you try `--baseURL https://pismice.github.io/HEIG_ZIG`

**Reply by Pismice** [reply]
that was good thank you :) But now for some unkown reason without modifying any config file I cant even acess basic things like the svg logo, no idea where they are. I dont know if this is related to this issue: https://pismice.github.io/HEIG_ZIG/docs/introduction/ any idea ?

**Reply by Pismice** [reply]
Might seem weird but would you be ok if I show you my problem on discord really fast ?
